internal/namespaces/instance/v1/custom_server_create_test.go

```diff
@@ -485,3 +486,29 @@ func Test_CreateServerErrors(t *testing.T) {
 		),
 	}))
 }
+
+func Test_CreateServerScratchStorage(t *testing.T) {
+	t.Run("Default scratch storage", core.Test(&core.TestConfig{
+		Commands: GetCommands(),
+		Cmd:      "scw instance server create type=H100-1-80G image=ubuntu_jammy_gpu_os_12 --debug",
```

Comment:
is it normal to have the debug flag here?

Reply:
It was for debugging purpose, requests are not logged unless this is added